### ISSUE TITLE
install_dependencies.py broken (gstreamer) (Trac #111)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#115]
The install_dependencies.py script is relying on the OSSBUILD_GSTREAMER_DIR environment variable to configure gstreamer, but the environment variable is not available until after the shell (or computer) is restarted.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/111
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "The install_dependencies.py script is relying on the OSSBUILD_GSTREAMER_DIR environment variable to configure gstreamer, but the environment variable is not available until after the shell (or computer) is restarted.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "install_dependencies.py broken (gstreamer)", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-07-14T15:41:53", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120728T02:47:43: 
* ryan@fobel.net commented:

> commit 8c2f79a697df76803dc016c130086cd2f86977a9
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Fri Jul 27 22:16:41 2012 -0400
> 
>     Fix installer/gstreamer dll issues
>     
>     Create a new binary dependencies repository. Fixes #115, #116.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
